reference/commands.rst

```diff
@@ -178,7 +178,7 @@ conan user
 
 	$ conan user [-p PASSWORD] [--remote REMOTE] [name]
 
-Update your cached user [and your password] to avoid it will be requested later, e.g., while you're uploading a package.
-You can have more than one user, and locally manage all your packages from your different accounts
+Update your cached user name [and password] to avoid it being requested later, e.g. while you're uploading a package.
+You can have more than one users, and locally manage all your packages from your different accounts,
```

Comment:
You're probably right. I just changed that without giving it much thought. Sorry if it's wrong /:)